contrib_bots/lib/sched.py

```diff
@@ -0,0 +1,186 @@
+# pip install schedule is necessary
+import schedule
+import time
+import simplejson as json
+import threading
+from random import randint
+
+class SchedHandler(object):
+    '''
+    This bot allows you to schedule daily alerts. It
+    looks for messages starting with '@every' or '@cancel' or '@recover'.
+
+    In this example, we write alerts at certain time intervals
+    in a new stream called "alert,".
+    '''
+
+    def usage(self):
+        def run_threaded(job_func):
+            job_thread = threading.Thread(target=job_func)
+            job_thread.start()
+
+        def event_handler():
+            time.sleep(5)
+            schedule.run_pending()
+
+        schedule.every(1).seconds.do(run_threaded, event_handler)
```

Comment:
I think these durations (5, 1, 2) should be configurable with the configuration file.